.circleci/config.yml

```diff
@@ -0,0 +1,89 @@
+version: 2
```

Comment:
Is this branch definitely up to date with master? It shouldn’t be showing any changes in the circle file right? Or am I getting confused in my old age? 🙈

Reply:
Should be, I just pulled master in this branch and it seems fine :D Literally just did it because I wasn't sure 😆